tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -427,6 +427,42 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system"
 	},
 
+	/** Firmware layout for the TL-WA850RE v2 */
+	{
+		.id     = "TLWA850REV2",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:55530000}\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:00000000}\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:55534100}\n",
+		.support_trail = '\x00',
```

Comment:
@heinzek, done, thanks! I have also fixed commit message as is turned out that there is no way to access the device over SSH in vendor firmware.